package.json

```
@@ -315,7 +315,7 @@
     "electron-builder": "26.0.19",
     "eslint": "^9.36.0",
     "eslint-config-airbnb-typescript": "^18.0.0",
-    "eslint-config-expensify": "2.0.100",
+    "eslint-config-expensify": "^2.0.101",
```

Whoops, please don't use the `^`:


```suggestion
    "eslint-config-expensify": "2.0.101",
```

We don't use proper semver in most of our dependencies, meaning that we almost never want `^` because it makes it so that we might unintentionally upgrade the dependency to anything less than `3.0`. I don't understand why this is such a popular practice, as it also just makes it much less clear what package version is actually installed